test/testdata/infer/generics/has_attached_class_include.rb

```diff
@@ -11,7 +11,7 @@ module ChildModule1 # error: `has_attached_class!` declared by parent `Parent` m
   include Parent
 end
 
-module ChildModule2 # error: `Parent` was declared `has_attached_class!` and so cannot be `extend`ed into the module `ChildModule2
+module ChildModule2 # error: Type variance mismatch for `T.attached_class` with parent `Parent`. Parent needs to be `:out` but it is `invariant`
```

Comment:
This is an unfortunate error regression, is there any way to recover the old error?

Reply:
Reading the comment in `GlobalPass.cc`, it's not clear that there's a good way to recover this now.

Reply:
Yeah. It's not great but unfortunately I don't think we can do better.

After I land this stack, I will be sure to add a note to the website that it's basically impossible to use `has_attached_class!` invariantly and also correctly.